cypress/integration/manualTests/auth-tests.ts

```diff
@@ -0,0 +1,251 @@
+import { goToTab } from '../../support/commands';
+
+// tuples of registry, repo, name
+const username = 'dockstoretestuser4';
+const toolTuple = ['github.com', username, 'dockstore-tool-md5sum'];
+const workflowTuple = ['github.com', username, 'hello-dockstore-workflow'];
+// tuple of organization name, collection name
+const collectionTuple = ['test', 'testcollection'];
+
+// get the dockstore token from env variable and put it in local storage
+function storeToken() {
+  window.localStorage.setItem('ng2-ui-auth_token', Cypress.env('TOKEN'));
+}
+
+function unpublishTool() {
+  it('unpublish the tool', () => {
+    storeToken();
+    cy.contains('button', 'Unpublish').should('be.visible');
+    cy.contains('button', 'Unpublish').click();
+  });
+}
+
+function deleteTool() {
+  it('delete the tool', () => {
+    cy.contains('button', 'Delete').should('be.visible');
+    cy.contains('button', 'Delete').click();
+    cy.contains('div', 'Are you sure you wish to delete this tool?').within(() => {
+      cy.contains('button', 'Delete').click();
+    });
+    cy.contains('There are currently no tools registered under this account');
+  });
+}
+
+function registerQuayTool(repo: string, name: string) {
+  it('register and publish - quickly register quay.io tools option', () => {
+    storeToken();
+
+    // define routes to watch for
+    cy.server();
+    cy.route('/api/users/dockerRegistries/quay.io/organizations').as('orgs');
+    cy.route('**/tokens').as('tokens');
+    cy.route('**/repositories').as('repos');
+    cy.route('**/containers').as('containers');
+    cy.route('post', '**/publish').as('publish');
+
+    cy.visit('/my-tools');
+    // click thru the steps of registering a tool
+    cy.wait('@tokens');
+    // cy.get('#register_tool_button').should('be.visible');
```

Comment:
should code comment under what condition this would be un-commented or delete
ideally, an `if` with a documented boolean value so it gets compiled/doesn't deteriorate over time

Reply:
added comment explaining